docs/fsharp/style-guide/conventions.md

```diff
@@ -641,3 +641,19 @@ module Networking =
 ```
 
 In summary, the pitfall with Type Abbreviations is that they are **not** abstractions over the types they are abbreviating. In the previous example, `BufferSize` is just an `int` under the covers, with no additional data, nor any benefits from the type system besides what `int` already has.
+
+An alternative approach to using type abbreviations to represent a domain is to use single-case discriminated unions. The previous sample can be modeled as follows:
+
+```fsharp
+type BufferSize = BufferSize of int
+```
+
+If you write code that operats in terms of `BufferSize` and its underlying value, you need to construct one rather than pass in any arbitrary integer:
```

Comment:
```suggestion
If you write code that operates in terms of `BufferSize` and its underlying value, you need to construct one rather than pass in any arbitrary integer:
```